php/ext/google/protobuf/convert.c

```diff
@@ -349,10 +379,12 @@ bool Convert_PhpToUpb(zval* php_val, upb_MessageValue* upb_val, TypeInfo type,
       upb_val->int32_val = i64;
       return true;
     case kUpb_CType_UInt64:
-      if (!Convert_PhpToInt64(php_val, &i64)) {
+      uint64_t u64;
```

Comment:
This lines seems to break for PHP 8.0

Reply:
@mkruskal-google Hmm, why? 
Tested locally, build successful. Now I'm trying to find differences with CI. 

<details><summary>Build log</summary>
<p>

```
❯ php -v
PHP 8.0.30 (cli) (built: Sep  2 2023 08:05:13) ( NTS )
Copyright (c) The PHP Group
Zend Engine v4.0.30, Copyright (c) Zend Technologies
    with Zend OPcache v8.0.30, Copyright (c), by Zend Technologies

❯ phpize -v
Configuring for:
PHP Api Version:         20200930
Zend Module Api No:      20200930
Zend Extension Api No:   420200930

❯ php-config --version
8.0.30

❯ make clean
find . -name \*.gcno -o -name \*.gcda | xargs rm -f
find . -name \*.lo -o -name \*.o | xargs rm -f
find . -name \*.la -o -name \*.a | xargs rm -f
find . -name \*.so | xargs rm -f
find . -name .libs -a -type d|xargs rm -rf
rm -f libphp.la      modules/* libs/*
rm -f ext/opcache/jit/zend_jit_x86.c

❯ phpize
Configuring for:
PHP Api Version:         20200930
Zend Module Api No:      20200930
Zend Extension Api No:   420200930
configure.ac:22: warning: $as_echo is obsolete; use AS_ECHO(["message"]) instead
build/php.m4:2072: PHP_CONFIG_NICE is expanded from...
configure.ac:22: the top level
configure.ac:165: warning: The macro `AC_PROG_LIBTOOL' is obsolete.
configure.ac:165: You should run autoupdate.
build/libtool.m4:99: AC_PROG_LIBTOOL is expanded from...
configure.ac:165: the top level

❯ ./configure
checking for grep that handles long lines and -e... /usr/bin/grep
checking for egrep... /usr/bin/grep -E
checking for a sed that does not truncate output... /usr/bin/sed
checking for pkg-config... /usr/bin/pkg-config
checking pkg-config is at least version 0.9.0... yes
checking for cc... cc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether the compiler supports GNU C... yes
checking whether cc accepts -g... yes
checking for cc option to enable C11 features... none needed
checking how to run the C preprocessor... cc -E
checking for icc... no
checking for suncc... no
checking for system library directory... lib
checking if compiler supports -Wl,-rpath,... yes
checking build system type... x86_64-pc-linux-gnu
checking host system type... x86_64-pc-linux-gnu
checking target system type... x86_64-pc-linux-gnu
checking for PHP prefix... /usr
checking for PHP includes... -I/usr/include/php/20200930 -I/usr/include/php/20200930/main -I/usr/include/php/20200930/TSRM -I/usr/include/php/20200930/Zend -I/usr/include/php/20200930/ext -I/usr/include/php/20200930/ext/date/lib
checking for PHP extension directory... /usr/lib/php/20200930
checking for PHP installed headers prefix... /usr/include/php/20200930
checking if debug is enabled... no
checking if zts is enabled... no
checking for gawk... gawk
checking whether to enable Protobuf extension... yes, shared
checking how to print strings... printf
checking for a sed that does not truncate output... (cached) /usr/bin/sed
checking for fgrep... /usr/bin/grep -F
checking for ld used by cc... /usr/bin/ld
checking if the linker (/usr/bin/ld) is GNU ld... yes
checking for BSD- or MS-compatible name lister (nm)... /usr/bin/nm -B
checking the name lister (/usr/bin/nm -B) interface... BSD nm
checking whether ln -s works... yes
checking the maximum length of command line arguments... 1572864
checking how to convert x86_64-pc-linux-gnu file names to x86_64-pc-linux-gnu format... func_convert_file_noop
checking how to convert x86_64-pc-linux-gnu file names to toolchain format... func_convert_file_noop
checking for /usr/bin/ld option to reload object files... -r
checking for objdump... objdump
checking how to recognize dependent libraries... pass_all
checking for dlltool... no
checking how to associate runtime and link libraries... printf %s\n
checking for ar... ar
checking for archiver @FILE support... @
checking for strip... strip
checking for ranlib... ranlib
checking for gawk... (cached) gawk
checking command to parse /usr/bin/nm -B output from cc object... ok
checking for sysroot... no
checking for a working dd... /usr/bin/dd
checking how to truncate binary pipes... /usr/bin/dd bs=4096 count=1
checking for mt... mt
checking if mt is a manifest tool... no
checking for stdio.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for strings.h... yes
checking for sys/stat.h... yes
checking for sys/types.h... yes
checking for unistd.h... yes
checking for dlfcn.h... yes
checking for objdir... .libs
checking if cc supports -fno-rtti -fno-exceptions... no
checking for cc option to produce PIC... -fPIC -DPIC
checking if cc PIC flag -fPIC -DPIC works... yes
checking if cc static flag -static works... yes
checking if cc supports -c -o file.o... yes
checking if cc supports -c -o file.o... (cached) yes
checking whether the cc linker (/usr/bin/ld -m elf_x86_64) supports shared libraries... yes
checking whether -lc should be explicitly linked in... no
checking dynamic linker characteristics... GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... no
configure: patching config.h.in
configure: creating ./config.status
config.status: creating config.h
config.status: config.h is unchanged
config.status: executing libtool commands

❯ make -j32
/bin/bash /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/libtool --mode=compile cc -I. -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/include -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/main -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/usr/include/php/20200930 -I/usr/include/php/20200930/main -I/usr/include/php/20200930/TSRM -I/usr/include/php/20200930/Zend -I/usr/include/php/20200930/ext -I/usr/include/php/20200930/ext/date/lib  -DHAVE_CONFIG_H  -g -O2   -std=gnu99 -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/third_party/utf8_range -c /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/arena.c -o arena.lo 
/bin/bash /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/libtool --mode=compile cc -I. -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/include -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/main -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/usr/include/php/20200930 -I/usr/include/php/20200930/main -I/usr/include/php/20200930/TSRM -I/usr/include/php/20200930/Zend -I/usr/include/php/20200930/ext -I/usr/include/php/20200930/ext/date/lib  -DHAVE_CONFIG_H  -g -O2   -std=gnu99 -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/third_party/utf8_range -c /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/array.c -o array.lo 
/bin/bash /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/libtool --mode=compile cc -I. -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/include -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/main -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/usr/include/php/20200930 -I/usr/include/php/20200930/main -I/usr/include/php/20200930/TSRM -I/usr/include/php/20200930/Zend -I/usr/include/php/20200930/ext -I/usr/include/php/20200930/ext/date/lib  -DHAVE_CONFIG_H  -g -O2   -std=gnu99 -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/third_party/utf8_range -c /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/convert.c -o convert.lo 
/bin/bash /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/libtool --mode=compile cc -I. -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/include -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/main -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/usr/include/php/20200930 -I/usr/include/php/20200930/main -I/usr/include/php/20200930/TSRM -I/usr/include/php/20200930/Zend -I/usr/include/php/20200930/ext -I/usr/include/php/20200930/ext/date/lib  -DHAVE_CONFIG_H  -g -O2   -std=gnu99 -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/third_party/utf8_range -c /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/def.c -o def.lo 
/bin/bash /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/libtool --mode=compile cc -I. -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/include -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/main -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/usr/include/php/20200930 -I/usr/include/php/20200930/main -I/usr/include/php/20200930/TSRM -I/usr/include/php/20200930/Zend -I/usr/include/php/20200930/ext -I/usr/include/php/20200930/ext/date/lib  -DHAVE_CONFIG_H  -g -O2   -std=gnu99 -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/third_party/utf8_range -c /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/map.c -o map.lo 
/bin/bash /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/libtool --mode=compile cc -I. -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/include -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/main -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/usr/include/php/20200930 -I/usr/include/php/20200930/main -I/usr/include/php/20200930/TSRM -I/usr/include/php/20200930/Zend -I/usr/include/php/20200930/ext -I/usr/include/php/20200930/ext/date/lib  -DHAVE_CONFIG_H  -g -O2   -std=gnu99 -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/third_party/utf8_range -c /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/message.c -o message.lo 
/bin/bash /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/libtool --mode=compile cc -I. -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/include -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/main -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/usr/include/php/20200930 -I/usr/include/php/20200930/main -I/usr/include/php/20200930/TSRM -I/usr/include/php/20200930/Zend -I/usr/include/php/20200930/ext -I/usr/include/php/20200930/ext/date/lib  -DHAVE_CONFIG_H  -g -O2   -std=gnu99 -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/third_party/utf8_range -c /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/names.c -o names.lo 
/bin/bash /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/libtool --mode=compile cc -I. -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/include -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/main -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/usr/include/php/20200930 -I/usr/include/php/20200930/main -I/usr/include/php/20200930/TSRM -I/usr/include/php/20200930/Zend -I/usr/include/php/20200930/ext -I/usr/include/php/20200930/ext/date/lib  -DHAVE_CONFIG_H  -g -O2   -std=gnu99 -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/third_party/utf8_range -c /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/php-upb.c -o php-upb.lo 
/bin/bash /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/libtool --mode=compile cc -I. -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/include -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/main -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/usr/include/php/20200930 -I/usr/include/php/20200930/main -I/usr/include/php/20200930/TSRM -I/usr/include/php/20200930/Zend -I/usr/include/php/20200930/ext -I/usr/include/php/20200930/ext/date/lib  -DHAVE_CONFIG_H  -g -O2   -std=gnu99 -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/third_party/utf8_range -c /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/protobuf.c -o protobuf.lo 
/bin/bash /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/libtool --mode=compile cc -I. -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/include -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/main -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/usr/include/php/20200930 -I/usr/include/php/20200930/main -I/usr/include/php/20200930/TSRM -I/usr/include/php/20200930/Zend -I/usr/include/php/20200930/ext -I/usr/include/php/20200930/ext/date/lib  -DHAVE_CONFIG_H  -g -O2   -std=gnu99 -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/third_party/utf8_range -c /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/third_party/utf8_range/naive.c -o third_party/utf8_range/naive.lo 
/bin/bash /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/libtool --mode=compile cc -I. -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/include -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/main -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/usr/include/php/20200930 -I/usr/include/php/20200930/main -I/usr/include/php/20200930/TSRM -I/usr/include/php/20200930/Zend -I/usr/include/php/20200930/ext -I/usr/include/php/20200930/ext/date/lib  -DHAVE_CONFIG_H  -g -O2   -std=gnu99 -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/third_party/utf8_range -c /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/third_party/utf8_range/range2-neon.c -o third_party/utf8_range/range2-neon.lo 
/bin/bash /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/libtool --mode=compile cc -I. -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/include -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/main -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/usr/include/php/20200930 -I/usr/include/php/20200930/main -I/usr/include/php/20200930/TSRM -I/usr/include/php/20200930/Zend -I/usr/include/php/20200930/ext -I/usr/include/php/20200930/ext/date/lib  -DHAVE_CONFIG_H  -g -O2   -std=gnu99 -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/third_party/utf8_range -c /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/third_party/utf8_range/range2-sse.c -o third_party/utf8_range/range2-sse.lo 
libtool: compile:  cc -I. -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/include -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/main -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/usr/include/php/20200930 -I/usr/include/php/20200930/main -I/usr/include/php/20200930/TSRM -I/usr/include/php/20200930/Zend -I/usr/include/php/20200930/ext -I/usr/include/php/20200930/ext/date/lib -DHAVE_CONFIG_H -g -O2 -std=gnu99 -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/third_party/utf8_range -c /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/arena.c  -fPIC -DPIC -o .libs/arena.o
libtool: compile:  cc -I. -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/include -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/main -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/usr/include/php/20200930 -I/usr/include/php/20200930/main -I/usr/include/php/20200930/TSRM -I/usr/include/php/20200930/Zend -I/usr/include/php/20200930/ext -I/usr/include/php/20200930/ext/date/lib -DHAVE_CONFIG_H -g -O2 -std=gnu99 -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/third_party/utf8_range -c /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/array.c  -fPIC -DPIC -o .libs/array.o
libtool: compile:  cc -I. -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/include -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/main -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/usr/include/php/20200930 -I/usr/include/php/20200930/main -I/usr/include/php/20200930/TSRM -I/usr/include/php/20200930/Zend -I/usr/include/php/20200930/ext -I/usr/include/php/20200930/ext/date/lib -DHAVE_CONFIG_H -g -O2 -std=gnu99 -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/third_party/utf8_range -c /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/convert.c  -fPIC -DPIC -o .libs/convert.o
libtool: compile:  cc -I. -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/include -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/main -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/usr/include/php/20200930 -I/usr/include/php/20200930/main -I/usr/include/php/20200930/TSRM -I/usr/include/php/20200930/Zend -I/usr/include/php/20200930/ext -I/usr/include/php/20200930/ext/date/lib -DHAVE_CONFIG_H -g -O2 -std=gnu99 -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/third_party/utf8_range -c /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/protobuf.c  -fPIC -DPIC -o .libs/protobuf.o
libtool: compile:  cc -I. -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/include -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/main -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/usr/include/php/20200930 -I/usr/include/php/20200930/main -I/usr/include/php/20200930/TSRM -I/usr/include/php/20200930/Zend -I/usr/include/php/20200930/ext -I/usr/include/php/20200930/ext/date/lib -DHAVE_CONFIG_H -g -O2 -std=gnu99 -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/third_party/utf8_range -c /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/message.c  -fPIC -DPIC -o .libs/message.o
libtool: compile:  cc -I. -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/include -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/main -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/usr/include/php/20200930 -I/usr/include/php/20200930/main -I/usr/include/php/20200930/TSRM -I/usr/include/php/20200930/Zend -I/usr/include/php/20200930/ext -I/usr/include/php/20200930/ext/date/lib -DHAVE_CONFIG_H -g -O2 -std=gnu99 -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/third_party/utf8_range -c /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/php-upb.c  -fPIC -DPIC -o .libs/php-upb.o
libtool: compile:  cc -I. -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/include -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/main -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/usr/include/php/20200930 -I/usr/include/php/20200930/main -I/usr/include/php/20200930/TSRM -I/usr/include/php/20200930/Zend -I/usr/include/php/20200930/ext -I/usr/include/php/20200930/ext/date/lib -DHAVE_CONFIG_H -g -O2 -std=gnu99 -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/third_party/utf8_range -c /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/map.c  -fPIC -DPIC -o .libs/map.o
libtool: compile:  cc -I. -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/include -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/main -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/usr/include/php/20200930 -I/usr/include/php/20200930/main -I/usr/include/php/20200930/TSRM -I/usr/include/php/20200930/Zend -I/usr/include/php/20200930/ext -I/usr/include/php/20200930/ext/date/lib -DHAVE_CONFIG_H -g -O2 -std=gnu99 -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/third_party/utf8_range -c /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/def.c  -fPIC -DPIC -o .libs/def.o
libtool: compile:  cc -I. -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/include -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/main -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/usr/include/php/20200930 -I/usr/include/php/20200930/main -I/usr/include/php/20200930/TSRM -I/usr/include/php/20200930/Zend -I/usr/include/php/20200930/ext -I/usr/include/php/20200930/ext/date/lib -DHAVE_CONFIG_H -g -O2 -std=gnu99 -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/third_party/utf8_range -c /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/names.c  -fPIC -DPIC -o .libs/names.o
libtool: compile:  cc -I. -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/include -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/main -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/usr/include/php/20200930 -I/usr/include/php/20200930/main -I/usr/include/php/20200930/TSRM -I/usr/include/php/20200930/Zend -I/usr/include/php/20200930/ext -I/usr/include/php/20200930/ext/date/lib -DHAVE_CONFIG_H -g -O2 -std=gnu99 -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/third_party/utf8_range -c /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/third_party/utf8_range/naive.c  -fPIC -DPIC -o third_party/utf8_range/.libs/naive.o
libtool: compile:  cc -I. -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/include -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/main -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/usr/include/php/20200930 -I/usr/include/php/20200930/main -I/usr/include/php/20200930/TSRM -I/usr/include/php/20200930/Zend -I/usr/include/php/20200930/ext -I/usr/include/php/20200930/ext/date/lib -DHAVE_CONFIG_H -g -O2 -std=gnu99 -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/third_party/utf8_range -c /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/third_party/utf8_range/range2-neon.c  -fPIC -DPIC -o third_party/utf8_range/.libs/range2-neon.o
libtool: compile:  cc -I. -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/include -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/main -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/usr/include/php/20200930 -I/usr/include/php/20200930/main -I/usr/include/php/20200930/TSRM -I/usr/include/php/20200930/Zend -I/usr/include/php/20200930/ext -I/usr/include/php/20200930/ext/date/lib -DHAVE_CONFIG_H -g -O2 -std=gnu99 -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/third_party/utf8_range -c /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/third_party/utf8_range/range2-sse.c  -fPIC -DPIC -o third_party/utf8_range/.libs/range2-sse.o
/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/convert.c: In function ‘Convert_UpbToPhp’:
/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/convert.c:452:37: warning: format ‘%llu’ expects argument of type ‘long long unsigned int’, but argument 3 has type ‘uint64_t’ {aka ‘long unsigned int’} [-Wformat=]
  452 |         int size = sprintf(buf, "%llu", upb_val.uint64_val);
      |                                  ~~~^   ~~~~~~~~~~~~~~~~~~
      |                                     |          |
      |                                     |          uint64_t {aka long unsigned int}
      |                                     long long unsigned int
      |                                  %lu
/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/convert.c:452:38: warning: ‘__builtin___sprintf_chk’ may write a terminating nul past the end of the destination [-Wformat-overflow=]
  452 |         int size = sprintf(buf, "%llu", upb_val.uint64_val);
      |                                      ^
In file included from /usr/include/stdio.h:894,
                 from /usr/include/php/20200930/Zend/zend_portability.h:49,
                 from /usr/include/php/20200930/Zend/zend_types.h:25,
                 from /usr/include/php/20200930/Zend/zend.h:27,
                 from /usr/include/php/20200930/main/php.h:32,
                 from /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/convert.h:11,
                 from /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/convert.c:8:
/usr/include/x86_64-linux-gnu/bits/stdio2.h:38:10: note: ‘__builtin___sprintf_chk’ output between 20 and 21 bytes into a destination of size 20
   38 |   return __builtin___sprintf_chk (__s, __USE_FORTIFY_LEVEL - 1,
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   39 |                                   __glibc_objsize (__s), __fmt,
      |                                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   40 |                                   __va_arg_pack ());
      |                                   ~~~~~~~~~~~~~~~~~
/bin/bash /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/libtool --mode=link cc -shared -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/include -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/main -I/home/zlodes/projects/personal/protobuf/php/ext/google/protobuf -I/usr/include/php/20200930 -I/usr/include/php/20200930/main -I/usr/include/php/20200930/TSRM -I/usr/include/php/20200930/Zend -I/usr/include/php/20200930/ext -I/usr/include/php/20200930/ext/date/lib  -DHAVE_CONFIG_H  -g -O2    -o protobuf.la -export-dynamic -avoid-version -prefer-pic -module -rpath /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/modules  arena.lo array.lo convert.lo def.lo map.lo message.lo names.lo php-upb.lo protobuf.lo third_party/utf8_range/naive.lo third_party/utf8_range/range2-neon.lo third_party/utf8_range/range2-sse.lo 
libtool: link: cc -shared  -fPIC -DPIC  .libs/arena.o .libs/array.o .libs/convert.o .libs/def.o .libs/map.o .libs/message.o .libs/names.o .libs/php-upb.o .libs/protobuf.o third_party/utf8_range/.libs/naive.o third_party/utf8_range/.libs/range2-neon.o third_party/utf8_range/.libs/range2-sse.o    -g -O2   -Wl,-soname -Wl,protobuf.so -o .libs/protobuf.so
libtool: link: ( cd ".libs" && rm -f "protobuf.la" && ln -s "../protobuf.la" "protobuf.la" )
/bin/bash /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/libtool --mode=install cp ./protobuf.la /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/modules
libtool: install: cp ./.libs/protobuf.so /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/modules/protobuf.so
libtool: install: cp ./.libs/protobuf.lai /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/modules/protobuf.la
libtool: finish: PATH="/home/zlodes/.gvm/bin:/home/zlodes/.nvm/versions/node/v19.0.0/bin:/home/zlodes/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/home/zlodes/.local/share/JetBrains/Toolbox/scripts:/home/zlodes/.oh-my-zsh/custom/plugins/fzf-zsh-plugin/bin:/home/zlodes/.fzf/bin:/opt/mssql-tools18/bin:/usr/local/go/bin:/home/zlodes/go/bin:/home/zlodes/projects/personal/protobuf/php/node_modules/.bin:/home/zlodes/.config/composer/vendor/bin:/sbin" ldconfig -n /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/modules
----------------------------------------------------------------------
Libraries have been installed in:
   /home/zlodes/projects/personal/protobuf/php/ext/google/protobuf/modules

If you ever happen to want to link against installed libraries
in a given directory, LIBDIR, you must either use libtool, and
specify the full pathname of the library, or use the '-LLIBDIR'
flag during linking and do at least one of the following:
   - add LIBDIR to the 'LD_LIBRARY_PATH' environment variable
     during execution
   - add LIBDIR to the 'LD_RUN_PATH' environment variable
     during linking
   - use the '-Wl,-rpath -Wl,LIBDIR' linker flag
   - have your system administrator add LIBDIR to '/etc/ld.so.conf'

See any operating system documentation about shared libraries for
more information, such as the ld(1) and ld.so(8) manual pages.
----------------------------------------------------------------------

Build complete.
Don't forget to run 'make test'.
```

</p>
</details>